readme.txt

```diff
@@ -0,0 +1,56 @@
+=== Event Tickets Plus Extension: Attendee Data Editor ===
+Contributors: ModernTribe
+Donate link: http://m.tri.be/29
+Tags: events, tickets
+Requires at least: 4.5
+Tested up to: 4.9.6
+Requires PHP: 5.6
+Stable tag: 1.2.1
+License: GPL version 3 or any later version
+License URI: https://www.gnu.org/licenses/gpl-3.0.html
+
+Extends Event Tickets Plus' custom attendee field functionality, by making it posisble for site admins to edit submitted fields.
+
+== Description ==
+
+[Event Tickets Plus](https://theeventscalendar.com/product/wordpress-event-tickets-plus/) allows event organizers to [collect arbitrary data](https://theeventscalendar.com/knowledgebase/collecting-attendee-information/) from attendees. However, once submitted it is not possible or easy to edit it.
```

Comment:
> ...once submitted it is not possible or easy to edit it.
It's not possible or it's not easy?

How about just this (understanding that it's not editable... via UI... because _everything's_ editable via db):
> ...once submitted, it is not editable via wp-admin.

(Can't the users edit their own attendee meta?)